src/main/java/org/broadinstitute/hellbender/tools/copynumber/CondenseReadCounts.java

```diff
@@ -0,0 +1,203 @@
+                    100.00 *(skippedRecords/ (double)(skippedRecords + condensedRecords)),
+                    100.00 *(skippedReadCounts/ (skippedReadCounts + outputCounts))));
+
+            return "SUCCESS";
+        } catch (final IOException ex) {
+            throw new GATKException(ex.getMessage(), ex);
+        }
+    }
+
+    private SimpleCount composeRecord(final String contig, final int start, final int end, final int count) {
+        return new SimpleCount(new SimpleInterval(contig, start, end), count);
+    }
+
+    private class SimpleCountReader extends TableReader<SimpleCount> {
+        private SimpleCountReader(final String input) throws IOException {
+            super(input, new InputStreamReader(BucketUtils.openFile(input)));
+        }
+
+        private List<String> headerLines;
+        private boolean foundHeader = false;
+
+        @Override
+        public boolean isCommentLine(String[] line) {
+            return line.length > 0 &&
+                    (line[0].startsWith("#") || line[0].startsWith("@"));
+        }
+
+        @Override
+        public void processCommentLine(final String line, final long number) {
+            if (!foundHeader) {
+                if (headerLines == null) {
+                    headerLines = new ArrayList<>();
+                }
+                headerLines.add(line);
+            }
+        }
+
+        public List<String> headerLines() {
+            return headerLines == null ? Collections.emptyList() : Collections.unmodifiableList(headerLines);
+        }
+
+        @Override
+        public void processColumns(final TableColumnCollection inputColumns) {
+            if (!inputColumns.containsAll(SimpleCountCollection.SimpleCountTableColumn.COLUMNS.names())) {
+                throw formatException("missing columns");
+            }
+            foundHeader = true;
+        }
+
+        public SimpleCount createRecord(final DataLine dataLine) {
+            return SimpleCountCollection.SIMPLE_COUNT_RECORD_FROM_DATA_LINE_DECODER.apply(dataLine);
+        }
+    }
+
+    private static class SimpleCountWriter extends TableWriter<SimpleCount> {
+
+        private SimpleCountWriter(final String output) throws IOException {
+            super(composeWriter(output), SimpleCountCollection.SimpleCountTableColumn.COLUMNS);
+        }
+
+        private static OutputStreamWriter composeWriter(final String output) {
+            if (IOUtil.hasBlockCompressedExtension(output)) {
+                return new OutputStreamWriter(new BlockCompressedOutputStream(BucketUtils.createFile(output), (File) null));
+            } else {
+                return new OutputStreamWriter(BucketUtils.createFile(output));
+            }
+        }
+
+        public void writeHeaderComments(final List<String> comments) throws IOException {
+            for (final String line : comments) {
+                writeLineBypassingCSVWriter(line);
```

Comment:
Why is this necessary?

Reply:
Is to avoid "forced" quotation by CSVWriter of the @ header line ... that said ,looking deeper in CSVWriter code I might be able to use this backdoor.

Reply:
You create the underlying writer in this class and should already be able to use it without going through the TableWriter.  We do a similar thing in AbstractRecordCollection to write the SAM header.